goentri.com.stunning.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "stunning",
+  "serviceName": "Stunning",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+   "variableDescription": "cnamePointsTo is the URL to where the subdomain is pointed and cnameHost is the host ",
+  "sharedProviderName": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%cnameHost%",
```

Comment:
why not `@` host and work with `host` parameter?
Yes, in the end you have the same record created, but some providers (IONOS, GoDaddy AFAIK) are identifying unique templates based on quadruple (domain, host, providerId, serviceId). In this case if you apply a second template with just different `%cnameHost%` the first one will be just removed.

Reply:
Yeah, makes sense. I would also need to change this template anyway, I'll make all those changes soon.

Reply:
Thanks

Reply:
So you want to keep this a1 group with CNAME here?

Reply:
qq - in case they send both a Record and CNAME would we be able to use 2 different templates for the same request?

Reply:
2 templates in the same request are not possible in the protocol at the moment.
If you want to have both in the same request than this template is fine, with a downside of just one instance per domain. As you are setting A record on APEX I believe it's fine like this.

Reply:
Thanks @pawel-kow, I'm going to keep the template as it is.